dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/QueryOperator.java

```diff
@@ -51,6 +51,7 @@ public enum QueryOperator {
   IN("in", true),
   SW("sw"),
   EW("ew"),
+  EX("??"),
```

Comment:
Instead of question marks, we can just use "ex" for this?

Reply:
The question marks are part of PostgreSQL syntax. They are used in the query to filter JSONB entries based on the presence of specific keys. I can't change them to something else.